scripts/generate/mockgen.go

```diff
@@ -82,13 +73,22 @@ func main() {
 	goimports.Stdin = bytes.NewBufferString(withHeader)
 	goimports.Stdout = outputFile
 	err = goimports.Run()
-	outputFile.Close()
-	if err != nil {
-		fmt.Println(err)
-		os.Exit(1)
-	}
+	handleError(err)
 }
 
 func usage() {
 	fmt.Println(os.Args[0], " PACKAGE INTERFACE_NAMES OUTPUT_FILE")
 }
+
+// handleError captures and prints the error code and stderr if applicable
+func handleError(err error) {
+	if err == nil {
+		return
+	}
+	exitErr, ok := err.(*exec.ExitError)
+	if ok {
+		fmt.Fprintln(os.Stderr, string(exitErr.Stderr))
+	}
```

Comment:
Could you add a comment here, why this is needed, since the error has been printed below?